packages/react/src/Switch/Switch.tsx

```diff
@@ -0,0 +1,28 @@
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, InputHTMLAttributes, PropsWithChildren, useId } from 'react'
+
+export const Switch = forwardRef(
+  (
+    { className, id, ...restProps }: PropsWithChildren<InputHTMLAttributes<HTMLInputElement>>,
+    ref: ForwardedRef<HTMLInputElement>,
+  ) => {
+    const inputId = id || useId()
+
+    return (
+      <div className={clsx('amsterdam-switch', className)}>
+        <input
+          {...restProps}
+          className="amsterdam-switch__input"
+          id={inputId}
+          ref={ref}
+          type="checkbox"
+          role="switch"
+        />
+
+        <label className="amsterdam-switch__label" htmlFor={inputId}></label>
```

Comment:
@alimpens This label is not going to be filled as a formcontrol any time soon. Normally a switch is wrapped in a label in order to make it clickable while hiding the native checkbox. And before you say a it should be able to be used without a label. The label was already here.